### DISCUSSION TITLE
Is there a way we can disable all Controller inside a FormContext?

### DISCUSSION THREAD

**Discussion by ashah2193** [react-hook-form/react-hook-form#1945] [Q&A] [answered]
Hello, 

I am working on a form with nested components. We are building our UI using [Ant Design](https://ant.design/components/overview/). and to support it, we are using [Controller](https://react-hook-form.com/api#Controller) from react-hook-form.

As it's a long form, we are using [FormContext](https://react-hook-form.com/api#useFormContext).

Our requirement is to disable entire form based on one boolean flag. 

Is there any way that we can disable all from controls ? Like passing prop to useForm.

**Comment by bluebill1049** [comment, marked as answer]
sounds like just need to use React Context API to pass down the props, we don't have global disable functionality.

**Reply by ashah2193** [reply]
Is there a plan to add global disable functionality ?

**Reply by bluebill1049** [reply]
Don’t think.
